tests/jeos/imgsize.pm

```diff
@@ -0,0 +1,23 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run {
+    my $self   = shift;
+    my $iso    = get_var("HDD_1");
+    my $size   = -s $iso;
```

Comment:
Does this work? HDD_1 is relative to */var/lib/openqa/factory/hdd*, whereas working dir for worker is its pool.